apps/client/src/components/AddressDataStyles.js

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 import { themeColor } from "@datapunt/asc-ui";
 
-export const StyledAddressResult = styled.div`
+export const StyledAddressData = styled.div`
```

Comment:
Why did you rename this? Imo data could be anywhere in the application

Reply:
Will fix.